src/__stories__/Customization.stories.mdx

```diff
@@ -1,84 +1,267 @@
-            p={1}
-            m={1}
-            backgroundColor={color}
-          >
-            <p style={{ color: 'white' }}>{color}</p>
-          </Box>
-          <Box
-            textAlign="center"
-            width={100}
-            p={1}
-            m={1}
-            backgroundColor={color}
-          >
-            <p style={{ color: 'black' }}>{color}</p>
-          </Box>
-        </FlexBox>
-      ),
+Scaleway UI provide you a set of colors trough theme, those colors are defined by tokens following asana token pattern.
+
+Tokens are designed as follows: `sentiment.usageProminenceInteraction`
+
+Where:
+- **sentiment** = `primary`, `secondary`, `neutral`, `info`, `danger`, `warning` and `success`
+- **usage** = `text`, `background` and `border`.
+- **prominence** = `weak` and `strong`.
+- **interaction**: `disabled` and `hover`.
+
+<p>&nbsp;</p>
+
+### Simple example
+
+Let's see an example with an alert:
+
+<ThemeProvider theme={theme}>
+  <Global styles={[globalStyles]}/>
+  <Alert type="info">This is a simple alert with info sentiment</Alert>
+</ThemeProvider>
+
+**Text** and **icon** colors will be `theme.colors.info.text`<br/>
+**Background** color will be `theme.colors.info.background`
+
+In this example there are different colors, but they all are having the same sentiment: `info`.
+
+<p>&nbsp;</p>
+
+### Advanced example
+
+Easy right? Let's now make it a bit more complex adding prominence in our context, let's take another alert as example:
+
+<ThemeProvider theme={theme}>
+  <Global styles={[globalStyles]}/>
+  <Alert variant="filled" type="info">This is a simple alert with info sentiment and strong prominence</Alert>
+</ThemeProvider>
+
+**Text** and **icon** colors will be `theme.colors.info.textStrong`<br/>
+**Background** color will be `theme.colors.info.backgroundStrong`
+
+<ThemeProvider theme={theme}>
+  <Global styles={[globalStyles]}/>
+  <Alert variant="outlined" type="info" mt={2}>This is a simple alert with info sentiment and weak prominence</Alert>
+</ThemeProvider>
+
+**Text** and **icon** colors will be `theme.colors.info.textWeak`<br/>
+**Background** color will be `theme.colors.info.backgroundWeak`
+
+Here we have 2 alerts with different prominence, one is strong and the other one is weak.
+It results in a different sets of colors and style for expressing a different context on where the alert is displayed.
+
+<p>&nbsp;</p>
+
+### All colors
+
+Here are all colors that we use in Scaleway UI. You can customize them in your custom theme:
+
+<ThemeProvider theme={theme}>
+  <Global styles={[globalStyles]}/>
+  {Object.keys(localContrasts).map(
+    sentiment => (
+      <>
+        <Typography variant="title" style={{textTransform: 'capitalize'}}>
+          {sentiment}
+        </Typography>
+        <Row display="inline-flex" flexDirection="column" mr={2}>
+          {Object.keys(theme.colors[sentiment]).filter(context => context.includes('background'))?.map(context => (
+            <Box
```

Comment:
Use a div :)

Reply:
@chambo-e It's a bordered box 😅

Reply:
I've updated the PR with the new BorderedBox :)